src/CMakeLists.txt

```diff
@@ -934,20 +934,25 @@ configure_file(
 add_subdirectory(brag)
 
 # Everything you need to spin up a cluster with vstart.sh
-add_custom_target(vstart DEPENDS
+add_custom_target(vstart-base DEPENDS
```

Comment:
it's vstart-base not vstart-rados?

and ceph cli requires `cython_rados`, so maybe we can move cython_rados into vstart-rados (or vstart-base) ?